README.md

```diff
@@ -0,0 +1,9 @@
+# AeroGear Proposals
+This repository contains proposals, examples, PoCs for potential new mobile features to be build by the mobile team.
+
+## How to submit a proposal
+* Fork this repository 
+* Create a new directory with your proposal
```

Comment:
I am not getting the new dir for each proposal this just seems like noise. What is the benefit? Personally I would prefer somthing like
```
/cli
command_structure.md
goals_and_principles.md
/sdks
...
/services
...
```

Reply:
Because a proposal isn't required to be a single document.

For example if I am writing a proposal for a Android Studio plugin I would like to attach images which are mockups of what I am talking about.  The easiest way to keep them all together is to put them in a single directory.

Reply:
I see no issue with either adding those images to the same directory or creating a sub dir for assets

Reply:
Would we want to go ahead and create some top level directories then?

At first I was thinking that proposals would be very short lived, either getting shot down or turned into trello cards and such and then deleted.  If we want them to stick around then having more curation up front like you are proposing is probably a good idea.

Reply:
I think that proposals can be done as single file initially and after some time repo maintainers can just organize them into the structure they want. Less rules initially will be better.
Some proposals can be retired etc. We can manage that as we go.